File: uniexetask.core/Models/Timeline.cs
using System;
using System.Collections.Generic;

namespace uniexetask.core.Models;

public partial class Timeline
{
    public int TimelineId { get; set; }

    public string TimelineName { get; set; } = null!;

    public string Description { get; set; } = null!;

    public DateTime StartDate { get; set; }

    public DateTime EndDate { get; set; }
}

File: unitask.frontend/src/models/Timeline.js
class Timeline {
    constructor(timelineId, timelineName, description, startDate, endDate) {
        this.timelineId = timelineId;
        this.timelineName = timelineName;
        this.description = description;
        this.startDate = new Date(startDate);
        this.endDate = new Date(endDate);
    }
}

export default Timeline;